samples/net/sockets/big_http_download/Makefile.posix

```diff
@@ -0,0 +1,6 @@
+# This makefile builds the sample for a POSIX system, like Linux
```

Comment:
Yeah, I answered a similar comment in https://github.com/zephyrproject-rtos/zephyr/pull/5985#discussion_r166143530 . This is POSIX-compatible sample, and it uses standard POSIX tool, make, to build on a POSIX-compatible system. CMake won't bring anything new in terms of functionality, but will require quite an effort to convert. This can be done some time later, but would be very low priority IMHO.